moc_openstack_api_reporting.py

```diff
@@ -497,6 +497,7 @@ def process_compute_events(openstack_conn, script_datetime, openstack_data, clus
     last_run_datetime = datetime.datetime.fromisoformat(cluster_state["last_run_timestamp"])
 
     for server in openstack_data["server_dict"].values():
+        event_data_defined = 0
```

Comment:
Is there any reason in particular why this is an integer instead of a boolean? To me it feels like it should be a boolean `event_data_defined = True`. This makes the code easier to read and understand.

Reply:
Though in general, a better approach would be to define `event_data = dict()` before you start using it, as an empty dict. And then check for `if event_data:`. The conditional will return false for an empty_dict.